<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StructViewPathSettingsWidget</class>
 <widget class="QWidget" name="StructViewPathSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StructViewPathSettingsWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="KEditListBox" name="kcfg_StructureDefFiles">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="buttons">
      <set>KEditListBox::All</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <item row="0" column="0">
      <widget class="KPushButton" name="removeAllButton">
       <property name="text">
        <string>Remove all</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KPushButton" name="addFilesButton">
       <property name="text">
        <string>Add files...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KEditListBox</class>
   <extends>QGroupBox</extends>
   <header>keditlistbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
